assets/javascripts/kitten/components/popovers/call-to-action-popover.js

```diff
@@ -3,6 +3,25 @@ import { Popover } from 'kitten/components/popovers/popover'
 import { Button } from 'kitten/components/buttons/button'
 
 export class CallToActionPopover extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.handleCloseClick = this.handleCloseClick.bind(this)
+    this.getButtonClickHandler = this.getButtonClickHandler.bind(this)
+    this.renderButton = this.renderButton.bind(this)
+  }
+
+  handleCloseClick() {
+    if (!this.props.onCloseClick) return
+
+    return this.props.onCloseClick()
+  }
+
+  getButtonClickHandler(clickOptions) {
+    if (!clickOptions) return
+    if (clickOptions.closeOnClick) { return this.handleCloseClick }
```

Comment:
On ne pourrait pas retourner `this.props.onCloseClick` directement ici ? Ça éviterait d'avoir `handleCloseClick`.